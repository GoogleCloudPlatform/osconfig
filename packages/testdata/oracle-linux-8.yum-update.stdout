Last metadata expiration check: 0:34:22 ago on Mon 28 Apr 2025 03:31:47 PM UTC.
Dependencies resolved.
====================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
 Package                                                                                                                         Architecture                                                                                              Version                                                                                                                                   Repository                                                                                                                Size
====================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
Installing:
 kernel                                                                                                                          x86_64                                                                                                    4.18.0-553.50.1.el8_10                                                                                                                    ol8_baseos_latest                                                                                                         10 M
 kernel-core                                                                                                                     x86_64                                                                                                    4.18.0-553.50.1.el8_10                                                                                                                    ol8_baseos_latest                                                                                                         44 M
 kernel-devel                                                                                                                    x86_64                                                                                                    4.18.0-553.50.1.el8_10                                                                                                                    ol8_baseos_latest                                                                                                         24 M
 kernel-modules                                                                                                                  x86_64                                                                                                    4.18.0-553.50.1.el8_10                                                                                                                    ol8_baseos_latest                                                                                                         36 M
 kernel-uek                                                                                                                      x86_64                                                                                                    5.15.0-307.178.5.el8uek                                                                                                                   ol8_UEKR7                                                                                                                3.0 M
 kernel-uek-core                                                                                                                 x86_64                                                                                                    5.15.0-307.178.5.el8uek                                                                                                                   ol8_UEKR7                                                                                                                 62 M
 kernel-uek-devel                                                                                                                x86_64                                                                                                    5.15.0-307.178.5.el8uek                                                                                                                   ol8_UEKR7                                                                                                                 21 M
 kernel-uek-modules                                                                                                              x86_64                                                                                                    5.15.0-307.178.5.el8uek                                                                                                                   ol8_UEKR7                                                                                                                 71 M
 kernel-uek-modules-extra                                                                                                        x86_64                                                                                                    5.15.0-307.178.5.el8uek                                                                                                                   ol8_UEKR7                                                                                                                7.9 M
Upgrading:
 bpftool                                                                                                                         x86_64                                                                                                    5.15.0-307.178.5.el8uek                                                                                                                   ol8_UEKR7                                                                                                                3.8 M
 cpp                                                                                                                             x86_64                                                                                                    8.5.0-26.0.1.el8_10                                                                                                                       ol8_appstream                                                                                                             10 M
 device-mapper-multipath                                                                                                         x86_64                                                                                                    0.8.4-42.el8_10                                                                                                                           ol8_baseos_latest                                                                                                        209 k
 device-mapper-multipath-libs                                                                                                    x86_64                                                                                                    0.8.4-42.el8_10                                                                                                                           ol8_baseos_latest                                                                                                        334 k
 expat                                                                                                                           x86_64                                                                                                    2.2.5-17.0.1.el8_10                                                                                                                       ol8_baseos_latest                                                                                                        114 k
 freetype                                                                                                                        x86_64                                                                                                    2.9.1-10.el8_10                                                                                                                           ol8_baseos_latest                                                                                                        393 k
 gcc                                                                                                                             x86_64                                                                                                    8.5.0-26.0.1.el8_10                                                                                                                       ol8_appstream                                                                                                             23 M
 glibc                                                                                                                           x86_64                                                                                                    2.28-251.0.3.el8_10.16                                                                                                                    ol8_baseos_latest                                                                                                        2.2 M
 glibc-common                                                                                                                    x86_64                                                                                                    2.28-251.0.3.el8_10.16                                                                                                                    ol8_baseos_latest                                                                                                        1.0 M
 glibc-devel                                                                                                                     x86_64                                                                                                    2.28-251.0.3.el8_10.16                                                                                                                    ol8_baseos_latest                                                                                                         93 k
 glibc-gconv-extra                                                                                                               x86_64                                                                                                    2.28-251.0.3.el8_10.16                                                                                                                    ol8_baseos_latest                                                                                                        1.6 M
 glibc-headers                                                                                                                   x86_64                                                                                                    2.28-251.0.3.el8_10.16                                                                                                                    ol8_baseos_latest                                                                                                        498 k
 glibc-langpack-en                                                                                                               x86_64                                                                                                    2.28-251.0.3.el8_10.16                                                                                                                    ol8_baseos_latest                                                                                                        836 k
 gnutls                                                                                                                          x86_64                                                                                                    3.6.16-8.el8_10.3                                                                                                                         ol8_baseos_latest                                                                                                        1.0 M
 google-guest-agent                                                                                                              x86_64                                                                                                    1:20250327.01-g1.el8                                                                                                                      google-compute-engine                                                                                                     46 M
 google-osconfig-agent                                                                                                           x86_64                                                                                                    1:20250320.00-g1.el8                                                                                                                      google-compute-engine                                                                                                    7.6 M
 grub2-common                                                                                                                    noarch                                                                                                    1:2.02-164.0.2.el8_10                                                                                                                     ol8_baseos_latest                                                                                                        899 k
 grub2-efi-x64                                                                                                                   x86_64                                                                                                    1:2.02-164.0.2.el8_10                                                                                                                     ol8_baseos_latest                                                                                                        510 k
 grub2-tools                                                                                                                     x86_64                                                                                                    1:2.02-164.0.2.el8_10                                                                                                                     ol8_baseos_latest                                                                                                        2.0 M
 grub2-tools-efi                                                                                                                 x86_64                                                                                                    1:2.02-164.0.2.el8_10                                                                                                                     ol8_baseos_latest                                                                                                        488 k
 grub2-tools-extra                                                                                                               x86_64                                                                                                    1:2.02-164.0.2.el8_10                                                                                                                     ol8_baseos_latest                                                                                                        1.1 M
 grub2-tools-minimal                                                                                                             x86_64                                                                                                    1:2.02-164.0.2.el8_10                                                                                                                     ol8_baseos_latest                                                                                                        218 k
 iwl100-firmware                                                                                                                 noarch                                                                                                    999:39.31.5.1-999.39.el8                                                                                                                  ol8_baseos_latest                                                                                                        169 k
 iwl1000-firmware                                                                                                                noarch                                                                                                    999:39.31.5.1-999.39.el8                                                                                                                  ol8_baseos_latest                                                                                                        169 k
 iwl105-firmware                                                                                                                 noarch                                                                                                    999:18.168.6.1-999.39.el8                                                                                                                 ol8_baseos_latest                                                                                                        253 k
 iwl135-firmware                                                                                                                 noarch                                                                                                    999:18.168.6.1-999.39.el8                                                                                                                 ol8_baseos_latest                                                                                                        263 k
 iwl2000-firmware                                                                                                                noarch                                                                                                    999:18.168.6.1-999.39.el8                                                                                                                 ol8_baseos_latest                                                                                                        256 k
 iwl2030-firmware                                                                                                                noarch                                                                                                    999:18.168.6.1-999.39.el8                                                                                                                 ol8_baseos_latest                                                                                                        265 k
 iwl3160-firmware                                                                                                                noarch                                                                                                    999:25.30.13.0-999.39.el8                                                                                                                 ol8_baseos_latest                                                                                                        496 k
 iwl5000-firmware                                                                                                                noarch                                                                                                    999:8.83.5.1_1-999.39.el8                                                                                                                 ol8_baseos_latest                                                                                                        167 k
 iwl5150-firmware                                                                                                                noarch                                                                                                    999:8.24.2.2-999.39.el8                                                                                                                   ol8_baseos_latest                                                                                                        166 k
 iwl6000-firmware                                                                                                                noarch                                                                                                    999:9.221.4.1-999.39.el8                                                                                                                  ol8_baseos_latest                                                                                                        186 k
 iwl6000g2a-firmware                                                                                                             noarch                                                                                                    999:18.168.6.1-999.39.el8                                                                                                                 ol8_baseos_latest                                                                                                        239 k
 iwl6050-firmware                                                                                                                noarch                                                                                                    999:41.28.5.1-999.39.el8                                                                                                                  ol8_baseos_latest                                                                                                        193 k
 iwl7260-firmware                                                                                                                noarch                                                                                                    999:25.30.13.0-999.39.el8                                                                                                                 ol8_baseos_latest                                                                                                        5.7 M
 iwlax2xx-firmware                                                                                                               noarch                                                                                                    999:20250319-999.39.el8                                                                                                                   ol8_baseos_latest                                                                                                         28 M
 kernel-headers                                                                                                                  x86_64                                                                                                    4.18.0-553.50.1.el8_10                                                                                                                    ol8_baseos_latest                                                                                                         12 M
 kernel-tools                                                                                                                    x86_64                                                                                                    4.18.0-553.50.1.el8_10                                                                                                                    ol8_baseos_latest                                                                                                         11 M
 kernel-tools-libs                                                                                                               x86_64                                                                                                    4.18.0-553.50.1.el8_10                                                                                                                    ol8_baseos_latest                                                                                                         10 M
 kexec-tools                                                                                                                     x86_64                                                                                                    2.0.28-1.0.9.el8_10                                                                                                                       ol8_baseos_latest                                                                                                        537 k
 kpartx                                                                                                                          x86_64                                                                                                    0.8.4-42.el8_10                                                                                                                           ol8_baseos_latest                                                                                                        119 k
 libgcc                                                                                                                          x86_64                                                                                                    8.5.0-26.0.1.el8_10                                                                                                                       ol8_baseos_latest                                                                                                         93 k
 libgfortran                                                                                                                     x86_64                                                                                                    8.5.0-26.0.1.el8_10                                                                                                                       ol8_baseos_latest                                                                                                        656 k
 libgomp                                                                                                                         x86_64                                                                                                    8.5.0-26.0.1.el8_10                                                                                                                       ol8_baseos_latest                                                                                                        219 k
 libquadmath                                                                                                                     x86_64                                                                                                    8.5.0-26.0.1.el8_10                                                                                                                       ol8_baseos_latest                                                                                                        183 k
 libsmbclient                                                                                                                    x86_64                                                                                                    4.19.4-7.0.1.el8_10                                                                                                                       ol8_baseos_latest                                                                                                        156 k
 libstdc++                                                                                                                       x86_64                                                                                                    8.5.0-26.0.1.el8_10                                                                                                                       ol8_baseos_latest                                                                                                        483 k
 libtasn1                                                                                                                        x86_64                                                                                                    4.13-5.el8_10                                                                                                                             ol8_baseos_latest                                                                                                         76 k
 libwbclient                                                                                                                     x86_64                                                                                                    4.19.4-7.0.1.el8_10                                                                                                                       ol8_baseos_latest                                                                                                        129 k
 linux-firmware                                                                                                                  noarch                                                                                                    999:20250319-999.39.git430633ec.el8                                                                                                       ol8_baseos_latest                                                                                                        432 M
 linux-firmware-core                                                                                                             noarch                                                                                                    999:20250319-999.39.git430633ec.el8                                                                                                       ol8_baseos_latest                                                                                                        2.1 M
 microcode_ctl                                                                                                                   x86_64                                                                                                    4:20250211-1.0.1.el8_10                                                                                                                   ol8_baseos_latest                                                                                                         10 M
 perf                                                                                                                            x86_64                                                                                                    4.18.0-553.50.1.el8_10                                                                                                                    ol8_baseos_latest                                                                                                         13 M
 python3-perf                                                                                                                    x86_64                                                                                                    4.18.0-553.50.1.el8_10                                                                                                                    ol8_baseos_latest                                                                                                         11 M
 samba-client-libs                                                                                                               x86_64                                                                                                    4.19.4-7.0.1.el8_10                                                                                                                       ol8_baseos_latest                                                                                                        5.0 M
 samba-common                                                                                                                    noarch                                                                                                    4.19.4-7.0.1.el8_10                                                                                                                       ol8_baseos_latest                                                                                                        233 k
 samba-common-libs                                                                                                               x86_64                                                                                                    4.19.4-7.0.1.el8_10                                                                                                                       ol8_baseos_latest                                                                                                        182 k
 shim-x64                                                                                                                        x86_64                                                                                                    15.8-4.0.2.el8_10                                                                                                                         ol8_baseos_latest                                                                                                        483 k
 sos                                                                                                                             noarch                                                                                                    4.8.2-1.0.2.el8_10                                                                                                                        ol8_baseos_latest                                                                                                        964 k
 systemd                                                                                                                         x86_64                                                                                                    239-82.0.3.el8_10.5                                                                                                                       ol8_baseos_latest                                                                                                        3.7 M
 systemd-libs                                                                                                                    x86_64                                                                                                    239-82.0.3.el8_10.5                                                                                                                       ol8_baseos_latest                                                                                                        1.1 M
 systemd-pam                                                                                                                     x86_64                                                                                                    239-82.0.3.el8_10.5                                                                                                                       ol8_baseos_latest                                                                                                        516 k
 systemd-udev                                                                                                                    x86_64                                                                                                    239-82.0.3.el8_10.5                                                                                                                       ol8_baseos_latest                                                                                                        1.6 M
 tzdata                                                                                                                          noarch                                                                                                    2025b-1.0.1.el8                                                                                                                           ol8_baseos_latest                                                                                                        476 k

Transaction Summary
====================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================================
Install   9 Packages
Upgrade  64 Packages

Total download size: 939 M
