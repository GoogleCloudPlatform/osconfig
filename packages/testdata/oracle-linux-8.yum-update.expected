[]*packages.PkgInfo{
    &packages.PkgInfo{
        Name:    "kernel",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.18.0-553.50.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kernel-core",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.18.0-553.50.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kernel-devel",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.18.0-553.50.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kernel-modules",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.18.0-553.50.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kernel-uek",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "5.15.0-307.178.5.el8uek",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kernel-uek-core",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "5.15.0-307.178.5.el8uek",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kernel-uek-devel",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "5.15.0-307.178.5.el8uek",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kernel-uek-modules",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "5.15.0-307.178.5.el8uek",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kernel-uek-modules-extra",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "5.15.0-307.178.5.el8uek",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "bpftool",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "5.15.0-307.178.5.el8uek",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "cpp",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "8.5.0-26.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "device-mapper-multipath",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "0.8.4-42.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "device-mapper-multipath-libs",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "0.8.4-42.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "expat",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "2.2.5-17.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "freetype",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "2.9.1-10.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "gcc",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "8.5.0-26.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "glibc",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "2.28-251.0.3.el8_10.16",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "glibc-common",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "2.28-251.0.3.el8_10.16",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "glibc-devel",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "2.28-251.0.3.el8_10.16",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "glibc-gconv-extra",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "2.28-251.0.3.el8_10.16",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "glibc-headers",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "2.28-251.0.3.el8_10.16",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "glibc-langpack-en",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "2.28-251.0.3.el8_10.16",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "gnutls",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "3.6.16-8.el8_10.3",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "google-guest-agent",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "1:20250327.01-g1.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "google-osconfig-agent",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "1:20250320.00-g1.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "grub2-common",
        Arch:    "all",
        RawArch: "noarch",
        Version: "1:2.02-164.0.2.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "grub2-efi-x64",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "1:2.02-164.0.2.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "grub2-tools",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "1:2.02-164.0.2.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "grub2-tools-efi",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "1:2.02-164.0.2.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "grub2-tools-extra",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "1:2.02-164.0.2.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "grub2-tools-minimal",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "1:2.02-164.0.2.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl100-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:39.31.5.1-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl1000-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:39.31.5.1-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl105-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:18.168.6.1-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl135-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:18.168.6.1-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl2000-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:18.168.6.1-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl2030-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:18.168.6.1-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl3160-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:25.30.13.0-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl5000-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:8.83.5.1_1-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl5150-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:8.24.2.2-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl6000-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:9.221.4.1-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl6000g2a-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:18.168.6.1-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl6050-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:41.28.5.1-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwl7260-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:25.30.13.0-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "iwlax2xx-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:20250319-999.39.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kernel-headers",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.18.0-553.50.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kernel-tools",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.18.0-553.50.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kernel-tools-libs",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.18.0-553.50.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kexec-tools",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "2.0.28-1.0.9.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "kpartx",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "0.8.4-42.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libgcc",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "8.5.0-26.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libgfortran",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "8.5.0-26.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libgomp",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "8.5.0-26.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libquadmath",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "8.5.0-26.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libsmbclient",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.19.4-7.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libstdc++",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "8.5.0-26.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libtasn1",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.13-5.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "libwbclient",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.19.4-7.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "linux-firmware",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:20250319-999.39.git430633ec.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "linux-firmware-core",
        Arch:    "all",
        RawArch: "noarch",
        Version: "999:20250319-999.39.git430633ec.el8",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "microcode_ctl",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4:20250211-1.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "perf",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.18.0-553.50.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "python3-perf",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.18.0-553.50.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "samba-client-libs",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.19.4-7.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "samba-common",
        Arch:    "all",
        RawArch: "noarch",
        Version: "4.19.4-7.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "samba-common-libs",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "4.19.4-7.0.1.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "shim-x64",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "15.8-4.0.2.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "sos",
        Arch:    "all",
        RawArch: "noarch",
        Version: "4.8.2-1.0.2.el8_10",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "systemd",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "239-82.0.3.el8_10.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "systemd-libs",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "239-82.0.3.el8_10.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "systemd-pam",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "239-82.0.3.el8_10.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "systemd-udev",
        Arch:    "x86_64",
        RawArch: "x86_64",
        Version: "239-82.0.3.el8_10.5",
        Source:  packages.Source{},
    },
    &packages.PkgInfo{
        Name:    "tzdata",
        Arch:    "all",
        RawArch: "noarch",
        Version: "2025b-1.0.1.el8",
        Source:  packages.Source{},
    },
}